indigo_app/templates/indigo_api/document/_content.html

```diff
@@ -11,7 +11,7 @@
   <div class="document-sheet-container content-with-gutter">
     <div class="sheet-outer content">
       <div class="sheet-inner" id="document-sheet">
-        <div class="akoma-ntoso"></div>
+        <la-akoma-ntoso class="akoma-ntoso"></la-akoma-ntoso>
```

Comment:
Set the expression FRBR URI here, so that we have all of the properties to style against, not just the country. Use `document.expression_frbr_uri`

Reply:
done